cli/worker.rs

```diff
@@ -105,22 +105,22 @@ impl Worker {
     startup_data: StartupData,
     state: &Rc<State>,
   ) -> Self {
-    let mut isolate = deno_core::JsRuntime::new_with_loader(
-      state.clone(),
-      state.clone(),
-      startup_data,
-      false,
-    );
-
+    let mut isolate =
+      JsRuntime::new_with_loader(state.clone(), startup_data, false);
     {
       let global_state = state.global_state.clone();
-      let core_state_rc = JsRuntime::state(&isolate);
-      let mut core_state = core_state_rc.borrow_mut();
-      core_state.set_js_error_create_fn(move |core_js_error| {
+      let js_runtime_state = JsRuntime::state(&isolate);
+      let mut js_runtime_state = js_runtime_state.borrow_mut();
+      js_runtime_state.set_js_error_create_fn(move |core_js_error| {
```

Comment:
Nitpick: `get_error_class_fn` is set as a field, maybe we could use the same pattern for `js_error_create_fn`?

Reply:
Yes probably. I will leave this for future work, if you don't mind.